src/index.js

```diff
@@ -478,13 +479,10 @@ Plese note now GCP logging is truned off by default and should be turned on expl
         break
       }
       console.log('Depositing....')
-      // await this.api.fundDepositAndReserve(['all'], '5000000000', '500000000000000000')
-      // await this.api.fundDepositAndReserve(['orchestrators'], '1', '2')
-      // await this.api.fundDeposit(['broadcasters'], '1')
       console.log('Initialize round...', onames)
       await this.api.initializeRound([`${onames[0]}`])
       await wait(2000)
-      await this.api.fundDepositAndReserve(['broadcasters'], '2500000000', '1500000000')
+      await this.api.fundDepositAndReserve(['broadcasters'], '1000000000000000000000', '1000000000000000000000')
```

Comment:
What do you think about moving these concerns into the configuration file? Same with the amount passed to `fundAccount()`? Not sure if it adds value as hardcoding fairly large numbers does suffice for our purposes.

Reply:
You mean using the `Api` object provided by `src/api` to create a script that is executed in the callback for `th.run()` (similar to what is done in `examples/10broadcasters.js`)? We could do that, but if we don't change the standard setup behavior then any test-harness config file that specifies `standardSetup = true` would not allow for a payment workflow that works right out of the box once the nodes are spun up. I think its useful for the standard setup to allow everything to "just work" and then if we want to test specific non-standard behavior we can go into the config file and customize how the nodes are setup.

Reply:
> I think its useful for the standard setup to allow everything to "just work" 

That's true, thanks for clarifying !